packages/components/ng-log-to-customer/src/translations/Messages_fr_FR.json

```
@@ -0,0 +1,6 @@
+{
+  "clear_session": "Effacer la session",
+  "search_placeholder": "Rechercher un élément ?",
+  "list_no_logs": "En attente de logs...",
+  "error_get_url": "Une erreur a été rencontrée lors de la récupération des logs"
```

`Une erreur est survenue` is more common
```suggestion
  "error_get_url": "Une erreur est survenue lors de la récupération des logs"
```